arangod/VocBase/Methods/UpgradeTasks.cpp

```diff
@@ -317,40 +317,23 @@ Result createSystemStatisticsCollections(
     };
     std::vector<std::shared_ptr<VPackBuffer<uint8_t>>> buffers;
     Result res;
+    OperationOptions options(ExecContext::current());
```

Comment:
Do we really want to execute UpgradeTasks with the current user's permissions? 
We need to create collections here, potentially even in the `_system` database.
Shouldn't we run the UpgradeTasks with root permissions, always?

Reply:
Good point, this needs full access.
Adapted.